### PR TITLE
Remove reference to Travis in Contribution process

### PR DESCRIPTION
**Pull request by nevillelyh**
Trino uses GitHub Actions now. Remove reference to specific CI to be future proof.

### CODE REVIEW

**Review by mosabua — APPROVED** [review]

File: development/process.md [hunk 1]
```diff
@@ -59,4 +59,4 @@ reviews, impact on other contributors, and to keep the amount of rework from the
 </div>
 </div>
 </div>
-</div>
\ No newline at end of file
+</div>
```
Comment (martint) [inline development/process.md:62]: Unrelated change?

Comment (nevillelyh) [reply development/process.md:62]: Hmm, I think GH editor did that 🤷
